src/Core/src/ImageSources/IImageSourceService.cs

```diff
@@ -16,6 +16,11 @@ public interface IImageSourceService
 			IImageSource imageSource,
 			float scale = 1,
 			CancellationToken cancellationToken = default);
+#elif TIZEN || __TIZEN__
+		Task<IImageSourceServiceResult<bool>?> LoadImageAsync(
+			IImageSource imageSource,
+			Tizen.UIExtensions.ElmSharp.Image image,
+			CancellationToken cancellationToken = default);
```

Comment:
I notice the signature for this method does not return the actual image data or a representation of it?

This service is mainly for a way to go from the abstract `IImageSource` to `<native image>` and might be used in places where there is no view.

Is there a way to make this service return image data?

The concept of loading the image directly into the view is something we want to do, and we should do, for Android. However, I just did not start that yet simply because we got away without doing it. If Tizen needs to do this, or would prefer to load this way, then it might be better to create a new service.

```cs
interface IImageViewService
{
    Task<bool> LoadImage(IImageSource, NativeImageView);
}
```

We can then also add Android to that one as well.

I don't think we should change the way this particular service works. Instead we should add a new one - even if it is Tizen specific.

Reply:
https://github.com/dotnet/maui/issues/2271

That was the idea we wanted to do. Seems maybe android and Tizen have similar patterns.

Reply:
done!